bsp/boards/iot-lab_M3/spi.c

```diff
@@ -43,14 +43,14 @@ volatile spi_vars_t spi_vars;
 inline static void RESET_CLR(void) { GPIOC->BRR = 1<<1; }
 inline static void RESET_SET(void) { GPIOC->BSRR = 1<<1; }
 inline static void CSn_SET(void) { GPIOA->BSRR = 1<<4; }
-inline static void CSn_CLR(void) { GPIOA->BRR = 1<<4; }
+//inline static void CSn_CLR(void) { GPIOA->BRR = 1<<4; } //never referenced

```

Comment:
Got inspired by: https://github.com/openwsn-berkeley/openwsn-fw/blob/develop/bsp/boards/agilefox/spi.c#L47
Should I remove it then ?